crowbar_framework/app/controllers/barclamp_controller.rb

```diff
@@ -922,7 +922,8 @@ def proposal_update
           @proposal["attributes"][params[:barclamp]] = JSON.parse(params[:proposal_attributes])
           @proposal["deployment"][params[:barclamp]] = JSON.parse(params[:proposal_deployment])
           @service_object.save_proposal!(@proposal)
-          answer = @service_object.proposal_commit(params[:name])
+          # validation already happened on save
+          answer = @service_object.proposal_commit(params[:name], validate: false, validate_after_save: false)
```

Comment:
Again the dog is right.  But sounds like a great idea to de-duplicate validation!  How did you think of that?